### PR TITLE
fix submitting Back button on pressing Enter

### PR DESCRIPTION
**Pull request by PetrDlouhy**
## Description
If I press Enter on the two factor form, the Back button gets pressed. This fixes that.

## Screenshots (if appropriate):
Snímek obrazovky_2022-03-09_17-18-33

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: two_factor/templates/two_factor/_wizard_actions.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
      class="float-right btn btn-link">{% trans "Cancel" %}</a>
 {% endif %}
 {% if wizard.steps.prev %}
-  <button name="wizard_goto_step" type="submit"
+  <button name="wizard_goto_step" type="button"
```
Comment (claudep) [inline two_factor/templates/two_factor/_wizard_actions.html:8]: If you do that, clicking the button won't going back, unless you add a JS handler that do something when the button is clicked.

**Review by claudep — CHANGES_REQUESTED** [review]

File: two_factor/templates/two_factor/_wizard_actions.html (hunk 1 above)
Comment (dopry) [reply two_factor/templates/two_factor/_wizard_actions.html:8]: @PetrDlouhy do you have time or capacity to address @claudep's change request?

Comment (PetrDlouhy) [reply two_factor/templates/two_factor/_wizard_actions.html]: OK, @claudep is actually right. And it seems, that the first button is always default in HTML: https://stackoverflow.com/questions/1963245/multiple-submit-buttons-on-html-form-designate-one-button-as-default

So if we don't want to colide with user defined form styling, I am duplicating the submit button as hidden to the first place in the form.

**Comment by PetrDlouhy** [comment]
This seems to be working already with the current master. I am closing this PR.

**Pull request closed without merging**